mne/channels/montage.py

```diff
@@ -72,6 +74,10 @@ def __repr__(self):
              % (self.kind, len(self.ch_names), ', '.join(self.ch_names[:3])))
         return s
 
+    @property
+    def pos2d(self):
```

Comment:
don't use a property for something that does actual computation. Use a method call get_pos2d() and please add a test.